static/translations/messages_en.json

```diff
@@ -213,5 +213,13 @@
   "🚌 Commuting": "🚌 Commuting",
   "🤒 Out sick": "🤒 Out sick",
   "🌴 Vacationing": "🌴 Vacationing",
-  "🏠 Working remotely": "🏠 Working remotely"
+  "🏠 Working remotely": "🏠 Working remotely",
```

Comment:
nit: add items somewhere other than the end of the list, to avoid noise in the diff (because JSON doesn't permit a comma after the last item)